sys/od/Kconfig

```
@@ -0,0 +1,10 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_OD
+    bool "OD Hex Dump"
+    select MODULE_FMT
```

Only available when testing module dependency modelling:
```suggestion
config MODULE_OD
    bool "OD Hex Dump"
    select MODULE_FMT
    depends on TEST_KCONFIG
```